File: app/src/main/java/com/privat/contacts/presentation/details/model/UserDetailsUi.java
package com.privat.contacts.presentation.details.model;

import android.widget.ImageView;
import android.widget.TextView;

import com.google.android.material.floatingactionbutton.FloatingActionButton;

import org.jetbrains.annotations.NotNull;

public interface UserDetailsUi {
    void showName(@NotNull TextView textView);

    void showPhone(@NotNull TextView textView);

    void loadAvatar(@NotNull ImageView imageView);

    void showFavorite(@NotNull FloatingActionButton floatingActionButton);
}

File: app/src/main/java/com/privat/contacts/presentation/details/model/UserDetailsUi.kt
package com.privat.contacts.presentation.details.model

import android.widget.ImageView
import android.widget.TextView
import com.google.android.material.floatingactionbutton.FloatingActionButton

interface UserDetailsUi {
    fun showName(textView: TextView)
    fun showPhone(textView: TextView)
    fun loadAvatar(imageView: ImageView)
    fun showFavorite(floatingActionButton: FloatingActionButton)
}
